d2l-table-style.js

```diff
@@ -1,45 +1,33 @@
+import '@brightspace-ui/core/components/colors/colors.js';
 import '@polymer/polymer/polymer-legacy.js';
-import 'd2l-colors/d2l-colors.js';
-import './d2l-table-shared-styles.js';
+import { tableStyles } from '@brightspace-ui/core/components/table/table-wrapper.js';
+
+if (document.getElementById('d2l-table-style') === null) {
+	const style = document.createElement('style');
+	style.id = 'd2l-table-style';
+	style.appendChild(document.createTextNode(tableStyles.cssText));
+	document.head.appendChild(style);
+}
+
 const $_documentContainer = document.createElement('template');
 
 $_documentContainer.innerHTML = `<dom-module id="d2l-table-style">
 	<template>
 		<style>
-			.d2l-table {
-				background-color: transparent;
-				border-collapse: separate!important;
-				border-spacing: 0;
-				display: table;
-				font-size: 0.8rem;
-				font-weight: 400;
-				width: 100%;
-			}
```

Comment:
I've removed the `<table class="d2l-table">`-specific CSS from the legacy style module since it's redundant with the Lit styles imported above. If both are present it actually causes issues since the Lit styles use a bottom-border approach vs. these styles use top borders.

The `d2l-table`/`d2l-tr`/etc. styles are the only ones that remain in here.